fs-api/src/System/FS/CRC.hs

```diff
@@ -1,76 +1,107 @@
-{-# LANGUAGE BangPatterns               #-}
-{-# LANGUAGE DeriveGeneric              #-}
+{-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE FlexibleInstances #-}
 {-# LANGUAGE GeneralizedNewtypeDeriving #-}
-{-# LANGUAGE ScopedTypeVariables        #-}
+{-# LANGUAGE MagicHash #-}
+{-# LANGUAGE ScopedTypeVariables #-}
 
 -- | Support for CRC
 module System.FS.CRC (
     -- * Wrap digest functionality
-    CRC (..)
-  , computeCRC
-  , initCRC
-  , updateCRC
+    CRC (..),
+    computeCRC,
+    initCRC,
+    updateCRC,
```

Comment:
`updateCRC` and `computeCRC` reference th enew `CRC32C` class but the class isn't exposed. Do we really need a new class?